src/Microsoft.Health.SqlServer/Features/Schema/Manager/SchemaClient.cs

```diff
@@ -17,94 +18,114 @@ namespace Microsoft.Health.SqlServer.Features.Schema.Manager
 {
     public class SchemaClient : ISchemaClient, IDisposable
     {
-        private HttpClient _httpClient;
+        private readonly HttpClient _httpClient;
         private const string Slash = "/";
 
         public SchemaClient()
         {
             _httpClient = new HttpClient();
         }
 
-        public void SetUri(Uri serverUrl)
+        public void SetUri(Uri uri)
         {
             if (_httpClient.BaseAddress == null)
             {
-                _httpClient.BaseAddress = serverUrl;
+                _httpClient.BaseAddress = uri;
             }
         }
 
         public async Task<List<CurrentVersion>> GetCurrentVersionInformationAsync(CancellationToken cancellationToken)
         {
-            var currentUrl = Slash + KnownRoutes.SchemaRoot + Slash + KnownRoutes.Current;
-            var response = await _httpClient.GetAsync(RelativeUrl(currentUrl), cancellationToken);
+            const string currentUrl = Slash + KnownRoutes.SchemaRoot + Slash + KnownRoutes.Current;
```

Comment:
Field?

Reply:
Alternatively, why don't we just build these up in `KnownRoutes.cs`?

Reply:
Moved over to `KnownRoutes`